src/app/public/modules/numeric/numeric.service.ts

```diff
@@ -82,12 +82,11 @@ export class SkyNumericService {
       // Note: This will need to be reviewed if we support currencies with
       // three decimal digits.
       case 'currency':
-        const isShortened = (value > this.symbolIndex[this.symbolIndex.length - 1].value);
         const isDecimal = (value % 1 !== 0);
 
         if (options.minDigits) {
           digits = `1.${options.minDigits}-${options.digits}`;
-        } else if (!isShortened && isDecimal && options.digits >= 2) {
```

Comment:
☝️I may be missing something, but the `isShortened` logic didn't seem to be doing anything? I tested the use case written above in the comments and double checked all unit tests were still passing. This logic seemed to force any number over `1000` to never allow 2 decimal places.